cardano-node/README.md

```diff
@@ -101,7 +101,7 @@ A CLI utility to support a variety of key material operations (genesis, migratio
 
 The general synopsis is as follows:
  ```
-   Usage: cardano-cli (Genesis related CMDs | Key related CMDs | Delegation related CMDs | Transaction related CMDs | Local node related CMDs | Benchmarking related CMDs)
+   Usage: cardano-cli (Byron related commands |Genesis related CMDs | Key related CMDs | Delegation related CMDs | Transaction related CMDs | Local node related CMDs | Benchmarking related CMDs)
```

Comment:
Nitpick, but these all say "CMDs" so you could either change all of them to "commands" or use "CMDs".

```suggestion
   Usage: cardano-cli (Byron related CMDs | Genesis related CMDs | Key related CMDs | Delegation related CMDs | Transaction related CMDs | Local node related CMDs | Benchmarking related CMDs)
```

Unless this is the actual output from the executable, in which case nevermind.